Documentation/subcommands/prepare.md

```diff
@@ -0,0 +1,32 @@
+# rkt prepare
+
+rkt can prepare images to run in a pod. This means it will fetch (if necessary) the images, extract them in its internal tree store, and allocate a pod UUID.
```

Comment:
+"If overlay fs is not supported or is disabled, it will also copy the tree in the pod rootfs."